config/routes.rb

```diff
@@ -304,10 +297,91 @@
   get "signup" => "static#show", id: "signup", constraints: { format: /(json|html)/ }
   get "login-preferences" => "static#show", id: "login", constraints: { format: /(json|html)/ }
 
+  get "my/*path", to: 'users#my_redirect'
+  get "user_preferences" => "users#user_preferences_redirect"
+
+  # New /u/ routes
```

Comment:
Oh actually I spoke too fast. Removing the `as: whatever` is a breaking change because those path helpers are no longer present in the app. Even if our tests pass, we can't count on plugins not having them. I'm going to merge without this for now, but if you want to figure out a solution for the `as` thing in a follow up commit that's cool. (maybe only add `as` if the path is `/u/`?)